keyboards/mode/m256wh/info.json

```diff
@@ -0,0 +1,107 @@
+{
+    "manufacturer": "Mode Designs",
+    "keyboard_name": "M256W-H",
+    "url": "",
+    "maintainer": "Gondolindrim",
+    "usb": {
+        "vid": "0x00DE",
+        "pid": "0x5754",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "lto": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true
+    },
+   "matrix_pins": {
+       "cols": ["C9" , "B1" , "B0" , "A10", "A15", "C10", "C11", "D2" , "B3" , "B4" , "B8" , "B9", "C0" , "C1" , "C2" , "C4" ],
+       "rows": ["C6" , "C7" , "C8" , "C12", "A7" , "C5" ]
+   },
+   "rgblight": {
+       "led_count": 30,
+       "pin": "B15"
```

Comment:
This will need to be changed to ws2812.pin on develop.

Reply:
Indeed, but when is the next devmerge?

Reply:
end of may. 

Also, mostly posting for visibility.  No action needs to be taken.  Eg, when it gets merged, can follow up with a PR to fix.